catalog/core/catalog-core-standardframework/src/main/java/ddf/catalog/impl/operations/QueryOperations.java

```diff
@@ -174,53 +132,15 @@ QueryResponse query(QueryRequest queryRequest, FederationStrategy strategy,
                 }
             }
 
-            queryResponse = doQuery(queryReq, fedStrategy, fanoutEnabled);
-
+            queryResponse = doQuery(queryRequest, fedStrategy, fanoutEnabled);
             queryResponse = injectAttributes(queryResponse);
-
-            validateFixQueryResponse(queryResponse, queryReq, overrideFanoutRename, fanoutEnabled);
-
-            HashMap<String, Set<String>> responsePolicyMap = new HashMap<>();
-            unmodifiableProperties = Collections.unmodifiableMap(queryResponse.getProperties());
-            for (Result result : queryResponse.getResults()) {
-                HashMap<String, Set<String>> itemPolicyMap = new HashMap<>();
-                for (PolicyPlugin plugin : frameworkProperties.getPolicyPlugins()) {
-                    try {
-                        PolicyResponse policyResponse = plugin.processPostQuery(result,
-                                unmodifiableProperties);
-                        opsSecuritySupport.buildPolicyMap(itemPolicyMap,
-                                policyResponse.itemPolicy()
-                                        .entrySet());
-                        opsSecuritySupport.buildPolicyMap(responsePolicyMap,
-                                policyResponse.operationPolicy()
-                                        .entrySet());
-                    } catch (StopProcessingException e) {
-                        throw new FederationException("Query could not be executed.", e);
-                    }
-                }
-                result.getMetacard()
-                        .setAttribute(new AttributeImpl(Metacard.SECURITY, itemPolicyMap));
-            }
-            queryResponse.getProperties()
-                    .put(PolicyPlugin.OPERATION_SECURITY, responsePolicyMap);
-
-            for (AccessPlugin plugin : frameworkProperties.getAccessPlugins()) {
-                try {
-                    queryResponse = plugin.processPostQuery(queryResponse);
-                } catch (StopProcessingException e) {
-                    throw new FederationException("Query could not be executed.", e);
-                }
-            }
-
-            for (PostQueryPlugin service : frameworkProperties.getPostQuery()) {
-                try {
-                    queryResponse = service.process(queryResponse);
-                } catch (PluginExecutionException see) {
-                    LOGGER.warn("Error executing PostQueryPlugin: {}", see.getMessage(), see);
-                } catch (StopProcessingException e) {
-                    throw new FederationException("Query could not be executed.", e);
-                }
-            }
+            queryResponse = validateFixQueryResponse(queryResponse,
```

Comment:
Getting the returned, updated value from `validateFixQueryResponse` instead of ignoring it.